physics/GFS_GWD_generic.meta

```diff
@@ -145,6 +177,13 @@
   kind = kind_phys
   intent = in
   optional = F
+[gwd_generic_tend]
+  standard_name = true_if_GFS_GWD_generic_should_calculate_tendencies
```

Comment:
I believe the standard name should be something like flag_for_... @grantfirl once again your opinion would be appreciated.

Reply:
Agree: flag_for_generic_gwd_tendencies is better, although "generic" is a bit vague in this context. You mean generic as opposed to tendencies that Valery put in for the CIRES UGWP scheme?

Reply:
GFS_GWD_generic is the name of the scheme that calculates the tendencies if that flag is true